src/reducers/visualizations.js

```diff
@@ -1,29 +1,48 @@
+import { orObject } from '../util';
+import objectClean from 'd2-utilizr/lib/objectClean';
+
 /** @module reducers/visualizations */
 
+const isEmpty = p => p === undefined || p === null;
```

Comment:
Why not also use the d2-utilizr isEmpty function, as you did in the itemFilter reducer?

Reply:
Because it considers empty array to be empty and thus removes "interpretations": [ ] from the visualization object.